src/openzaak/components/documenten/tasks.py

```diff
@@ -0,0 +1,395 @@
+# SPDX-License-Identifier: EUPL-1.2
+# Copyright (C) 2019 - 2024 Dimpact
+import logging
+import shutil
+from pathlib import Path
+from uuid import UUID, uuid4
+
+from django.conf import settings
+from django.core.exceptions import ValidationError
+from django.db import Error as DatabaseError, IntegrityError, transaction
+from django.utils import timezone
+
+from rest_framework.test import APIRequestFactory
+from vng_api_common.constants import RelatieAarden
+from vng_api_common.utils import generate_unique_identification
+
+from openzaak import celery_app
+from openzaak.components.documenten.api.serializers import (
+    EnkelvoudigInformatieObjectSerializer,
+)
+from openzaak.components.documenten.import_utils import DocumentRow
+from openzaak.components.documenten.models import (
+    EnkelvoudigInformatieObject,
+    EnkelvoudigInformatieObjectCanonical,
+)
+from openzaak.components.zaken.models.zaken import Zaak, ZaakInformatieObject
+from openzaak.import_data.models import Import, ImportStatusChoices
+from openzaak.import_data.utils import (
+    finish_batch,
+    finish_import,
+    get_csv_generator,
+    get_total_count,
+    task_locker,
+)
+from openzaak.utils.fields import get_default_path
+
+logger = logging.getLogger(__name__)
+
+
+def _import_document_row(
+    row: list[str],
+    row_index: int,
+    identifier: str,
+    existing_uuids: list[str],
+    zaak_uuids: dict[str, int],
+) -> DocumentRow:
+    expected_column_count = len(DocumentRow.import_headers)
+
+    request_factory = APIRequestFactory()
```

Comment:
I have an error after uploading csv file in celery container:

```
[2024-06-14 14:14:26,300: ERROR/ForkPoolWorker-4] Task openzaak.components.documenten.tasks.import_documents[8d584fcb-590a-4d45-8242-dcfd70dca187] raised unexpected: DisallowedHost("Invalid HTTP_HOST header: 'testserver'. You may need to add 'testserver' to ALLOWED_HOSTS.")
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/celery/app/trace.py", line 451, in trace_task
    R = retval = fun(*args, **kwargs)
  File "/usr/local/lib/python3.10/site-packages/celery/app/trace.py", line 734, in __protected_call__
    return self.run(*args, **kwargs)
  File "/app/src/openzaak/import_data/utils.py", line 207, in wrapped
    return func(*args, **kwargs)
  File "/app/src/openzaak/components/documenten/tasks.py", line 338, in import_documents
    document_row = _import_document_row(
  File "/app/src/openzaak/components/documenten/tasks.py", line 88, in _import_document_row
    if not eio_serializer.is_valid():
  File "/usr/local/lib/python3.10/site-packages/rest_framework/serializers.py", line 227, in is_valid
    self._validated_data = self.run_validation(self.initial_data)
  File "/usr/local/lib/python3.10/site-packages/rest_framework/serializers.py", line 426, in run_validation
    value = self.to_internal_value(data)
  File "/usr/local/lib/python3.10/site-packages/rest_framework/serializers.py", line 483, in to_internal_value
    validated_value = field.run_validation(primitive_value)
  File "/usr/local/lib/python3.10/site-packages/django_loose_fk/drf.py", line 159, in run_validation
    url = super().run_validation(*args, **kwargs)
  File "/usr/local/lib/python3.10/site-packages/rest_framework/fields.py", line 751, in run_validation
    return super().run_validation(data)
  File "/usr/local/lib/python3.10/site-packages/rest_framework/fields.py", line 548, in run_validation
    self.run_validators(value)
  File "/usr/local/lib/python3.10/site-packages/rest_framework/fields.py", line 560, in run_validators
    validator(value, self)
  File "/app/src/openzaak/utils/validators.py", line 134, in __call__
    super().__call__(value, serializer_field)
  File "/app/src/openzaak/utils/serializer_fields.py", line 68, in __call__
    super().__call__(url, serializer_field)
  File "/usr/local/lib/python3.10/site-packages/django_loose_fk/drf.py", line 88, in __call__
    host = serializer_field.context["request"].get_host()
  File "/usr/local/lib/python3.10/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: 'testserver'. You may need to add 'testserver' to ALLOWED_HOSTS.
```

I think it's because of this factory. Maybe it would be better to pass `request` to the celery task? Or at least the host